functions/Read-DbaTraceFile.ps1

```diff
@@ -168,7 +168,7 @@ function Read-DbaTraceFile {
         [int[]]$Spid,
         [string[]]$EventClass,
         [string[]]$ObjectType,
-        [int[]]$Error,
+        [int[]]$ErrorID,
```

Comment:
thanks so much! this should be $ErrorId, however, then we can merge 
```suggestion
        [int[]]$ErrorId,
```

Reply:
hey cool suggestion!

Reply:
That is a cool feature makes code reviews easier. Thanks for all the positive feed back on these PRs, You are doing a great job heading up this project!

Reply:
Thanks so much 🤗 I'm always happy to see these PRs coming in. I think it keeps the momentum going. Finally 1.0 seems possible!